Examples/Tests/particle_data_python/PICMI_inputs_2d.py

```diff
@@ -130,7 +130,7 @@ def add_particles():
         w=w, newPid=newPid, unique_particles=args.unique
     )
 
-callbacks.installbeforestep(add_particles)
+sim.extension.libwarpx_so.add_python_callback( "installbeforestep", add_particles )
```

Comment:
@ax3l @roelof-groenewald @dpgrote With the new Python wrappings, this seems to be a valid way to setup the callback functions, but is maybe a bit cumbersome (due to `sim.extension.libwarpx_so.`). Is there maybe a simpler way that I am overlooking?

Reply:
I just pushed a change to restore the old API.

Reply:
My thinking in refactoring things were that maybe we change the API to `callbacks.install('beforestep', add_particles)`, but we can decide on that later.

Reply:
Sounds good. Thanks!